src/garage/np/policies/policy_utils.py

```diff
@@ -0,0 +1,40 @@
+"""Module for utility functions commonly used by policies."""
+
+import numpy as np
+
+
+def flatten_if_unflattened(observation_space, observation):
+    """Flattens observation to a numpy.ndarray if it is not.
+
+    Args:
+        observation_space(akro.Space): The observation space that
+            observation comes from.
+        observation(observation_space.dtype): The observations to be flattened
+            if necessary.
+
+    Returns:
+        np.ndarray: Flattened observations.
+    """
+    if not isinstance(observation, np.ndarray):
+        observation = (observation_space.flatten(observation))
+    return observation
+
+
+def flatten_n_if_unflattened(observation_space, observations):
+    """Flattens a batch of observations to a numpy.ndarray if it is not.
+
+    Args:
+        observation_space(akro.Space): The observation space that
+            observations comes from.
+        observations(observation_space.dtype): The observations to be flattened
+            if necessary.
+
+    Returns:
+        np.ndarray: Flattened observations.
+    """
+    if isinstance(observations, list):
+        if isinstance(observations[0], np.ndarray):
```

Comment:
does this really tell me whether it's flat or not? what if the ndarray has more than 2 dimensions?

Reply:
Hmm. If the ndarray has more than 2 dimensions, we don't flatten it. There are still experiments that use 2d ndarray observations. We wouldn't want to flatten those because then they wouldn't match the flattened observation space. The reason why I added this if statement is because though it is documented that a user should pass numpy array observations, there were tests that passed lists of numpy arrays to the function `get_actions`